libraries/ms-common/inc/spi.h

```diff
@@ -0,0 +1,80 @@
+// HAL SPI
```

Comment:
It's nice to add a brief description of the module, what its purpose is, and what assumptions are made in the header.